.github/workflows/periodic.yml

```diff
@@ -19,11 +19,10 @@ jobs:
           node_image: kindest/node:v1.29.2
       - name: Run Functional Tests
         run: |
-          log_dir=$(mktemp -d)
```

Comment:
Please, add this line back and don't make any other changes. All we need is this line.

Reply:
oh, might have done it by mistake. I will add it back

Reply:
Done